### PR TITLE
settings: sub pages have no active state

### LINKED ISSUE #4505: settings: sub pages have no active state
https://user-images.githubusercontent.com/8019099/190510348-1faea00d-2dba-4041-becf-c042ea8f6632.mp4

should be same bg-color than the other active settings pages

### PR DESCRIPTION
**Pull request by gitstart**
## What does this PR do?

Change sidebar style

Fixes #4505

## Demo
https://www.loom.com/share/ac5375fa74ce43618d6cb4a61b1b93e7

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by gitstart** [comment]
Hi @PeerRich! Kindly review the PR

**Comment by PeerRich** [comment]
I believe the loom video expired

**Comment by gitstart** [comment]
> I believe the loom video expired

Sorry about this, will update with new loom video

**Comment by gitstart** [comment]
Hey @PeerRich! loom video has been updated now

File: packages/ui/v2/core/layouts/SettingsLayout.tsx [hunk 1]
```diff
@@ -319,7 +319,9 @@ export default function SettingsLayout({
     <Shell
       flexChildrenContainer
       {...rest}
-      SidebarContainer={<SettingsSidebarContainer className="hidden lg:flex" />}
+      SidebarContainer={
+        <SettingsSidebarContainer className="absolute inset-y-0 z-50 m-0 hidden h-screen translate-x-0 transform overflow-y-scroll border-gray-100 bg-gray-50 transition duration-200 ease-in-out lg:flex" />
```
Comment (PeerRich) [inline packages/ui/v2/core/layouts/SettingsLayout.tsx:323]: this adds so many classes and I don't know why we need them. can you explain?

Comment (Toledodev) [reply packages/ui/v2/core/layouts/SettingsLayout.tsx:323]: Hey @PeerRich,  I added these classes to side bar to have the exact style of the others, as shown in this comment https://github.com/calcom/cal.com/issues/4505

Comment (Toledodev) [reply packages/ui/v2/core/layouts/SettingsLayout.tsx:323]: Hey @PeerRich, I added these classes to side bar to have the exact style of the others, as shown in this comment https://github.com/calcom/cal.com/issues/4505, but when removing the changes I realized that these changes are no longer necessary
Loom video from main branch: https://www.loom.com/share/b0edde6090804c8ebd55f0a93d819e65

**Pull request closed without merging**